netbox/utilities/testing/views.py

```diff
@@ -583,6 +590,49 @@ def test_bulk_import_objects_with_permission(self):
             self.assertHttpStatus(self.client.post(self._get_url('import'), data), 200)
             self.assertEqual(self._get_queryset().count(), initial_count + len(self.csv_data) - 1)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=['*'])
+        def test_bulk_update_objects_with_permission(self):
+            if not self.csv_update_data:
+                raise NotImplementedError("The test must define csv_update_data.")
+
+            data = {
+                'csv': self._get_csv_data(),
+            }
+
+            # Assign model-level permission
+            obj_perm = ObjectPermission(
+                name='Test permission',
+                actions=['add']
+            )
+            obj_perm.save()
+            obj_perm.users.add(self.user)
+            obj_perm.object_types.add(ContentType.objects.get_for_model(self.model))
+
+            self.assertHttpStatus(self.client.post(self._get_url('import'), data), 200)
```

Comment:
This step shouldn't be necessary. We've already created some objects under `setUpTestData()` for each test, and we're modifying those. For instance, under ProviderTestCase, we have:

```python
        providers = (
            Provider(name='Provider 1', slug='provider-1'),
            Provider(name='Provider 2', slug='provider-2'),
            Provider(name='Provider 3', slug='provider-3'),
        )
        Provider.objects.bulk_create(providers)
```

and then:

```python
        cls.csv_update_data = (
            "id,name,comments",
            f"{providers[0].pk},Provider 7,New comment7",
            f"{providers[1].pk},Provider 8,New comment8",
            f"{providers[2].pk},Provider 9,New comment9",
        )
```

This will rename e.g. Provider 1 to Provider 7.